lbz/request.py

```diff
@@ -67,9 +67,7 @@ def _safe_json_loads(payload: Union[str, bytes]) -> Optional[Any]:
         try:
             return json.loads(payload)
         except ValueError as error:
-            raise BadRequestError(
-                "Invalid payload.\nPayload body:\n {!r}".format(payload)
-            ) from error
+            raise BadRequestError(f"Invalid payload.\nPayload body:\n {repr(payload)}") from error
```

Comment:
_FYI:_ I needed to fix the errors pointed out by `pylint` after the upgrade - you can check all the changes in a separate commit: [#e9f7bb9](https://github.com/pdyba/lambdalizator/pull/58/commits/e9f7bb91655a7d7d59ce94897120875775a243dd)